### ISSUE TITLE
[docs][self-request] Add Magnifying Glass Images for docs once magnifying glass PR lands

### ISSUE THREAD

**Issue by antholeole** [flutter/flutter#108041]
Chicken and egg problem: I want to add images for docs for the loupe, (like [this](https://github.com/flutter/assets-for-api-docs/commit/3333f4a2832912c1e80220e5e053290f6c764374), but more loupe and less bird) but since we need reproduceable code samples, so after https://github.com/flutter/flutter/pull/107477 lands, I can enrich the docs with images.

**Comment by antholeole** [comment]
Going to throw in the towel on this one :(. I've started some work but there's a lot of roadblocks. For some reason, you can only see one of the handles in the screenshot I'm generating. Another issue is that I'm using Offset.zero & MediaQuery.of(size) to determine the screen size bounding box, but that causes issues if the bounding box actually does not start at 0.

The work I have so far is [here](https://github.com/antholeole/assets-for-api-docs/tree/cupertino-magnifier) but this is such a low priority issue. I might remove the TODO in a PR on its own. :( 

cc @justinmc
